pixi.toml

```diff
@@ -0,0 +1,183 @@
+
+[dependencies]
+bleach = "*"
+bokeh = ">=3.4.0,<3.5.0"
+linkify-it-py = "*"
+markdown = "*"
+markdown-it-py = "*"
+mdit-py-plugins = "*"
+nodejs= ">=20"
+packaging = "*"
+pandas = ">=1.2"
+param = ">=2.1.0,<3.0"
+pip = "*"
+pyviz_comms = ">=2.0.0"
+requests = "*"
+tqdm = ">=4.48.0"
+typing_extensions = "*"
+# Recommended
+holoviews = ">=1.16.0"
+jupyterlab = "*"
+matplotlib = "*"
+pillow = "*"
+plotly = ">=4.0"
+watchfiles = "*"
+
+[feature.py39.dependencies]
+python = "3.9.*"
+
+[feature.py310.dependencies]
+python = "3.10.*"
+
+[feature.py311.dependencies]
+python = "3.11.*"
+
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.example.dependencies]
+aiohttp = "*"
+altair = "*"
+croniter = "*"
+dask-expr = "*"
+datashader = "*"
+fastparquet = "*"
+folium = "*"
+graphviz = "*"
+hvplot = "*"
+ipyleaflet = "<0.19" # tmp: https://github.com/conda-forge/ipyleaflet-feedstock/issues/116
+ipympl = "*"
+ipyvolume = "*"
+ipyvuetify = "*"
+ipywidgets = "*"
+ipywidgets_bokeh = "*"
+jupyter_bokeh = ">=3.0.7"
+networkx = ">=2.5"
+pydeck = "*"
+pyecharts = "*"
+pygraphviz = "*"
+pyinstrument = ">=4.0"
+python-graphviz = "*"
+pyvista = "*"
+reacton = "*"
+scikit-image = "*"
+scikit-learn = "*"
+seaborn = "*"
+streamz = "*"
+textual = "*"
+tqdm = ">=4.48.0"
+vega_datasets = "*"
+vtk = "*"
+xarray = "*"
+xgboost = "*"
+
+# =============================================
+# =================== TESTS ===================
+# =============================================
+[feature.test-core.dependencies]
+psutil = "*"
+pytest = "*"
+pytest-asyncio = "*"
+pytest-cov = "*"
+pytest-github-actions-annotate-failures = "*"
+pytest-rerunfailures = "*"
+pytest-xdist = "*"
+
+[feature.test.dependencies]
+altair = "*"
+anywidget = "*"
+diskcache = "*"
+folium = "*"
+ipympl = "*"
+ipyvuetify = "*"
+ipywidgets_bokeh = "*"
+numba = "*"
+reacton = "*"
+scipy = "*"
+textual = "*"
+
+[feature.test-unit-task.tasks] # So it is not showing up in the test-ui environment
+test-unit = 'pytest panel/tests -n logical --dist loadgroup'
+test-subprocess = 'pytest panel/tests --subprocess'
+
+[feature.test-example.tasks]
+test-docs = 'pytest panel/tests --docs'
+test-example = 'pytest -n logical --dist loadscope --nbval-lax examples'
+
+[feature.test-example.dependencies]
+nbval = "*"
+
+[feature.test-ui]
+channels = ["conda-forge", "pyviz/label/dev", "microsoft"]
+
+[feature.test-ui.dependencies]
+playwright = "*"
+pytest-playwright = "*"
+jupyter_server = "*"
+
+[feature.test-ui.tasks]
+x-install-ui = 'playwright install chromium'
+
+[feature.test-ui.tasks.test-ui]
+cmd = 'pytest panel/tests/ui --ui --browser chromium -n logical --dist loadgroup --reruns 3 --reruns-delay 10'
+depends_on = ["x-install-ui"]
+
+# =============================================
+# =================== DOCS ====================
+# =============================================
+[feature.doc.dependencies]
+lxml = "*"
+nbsite = ">=0.8.4"
+pandas = "<2.1.0" # Avoid deprecation warnings
```

Comment:
I think `lxml` and `pandas` are both pins we added to work-around errors/warnings.

Reply:
I have tried to remove them and build the docs:

https://github.com/holoviz/panel/actions/runs/8828880796

Reply:
The docs look fine. 

The only thing I can find about lxml is https://github.com/holoviz/panel/commit/8b0b7f7b44880d778e3fa5c80e7c94930957aaad

Reply:
`lxml` was needed as the docs could not run without it.